bosh-director/lib/bosh/director/jobs/ssh.rb

```diff
@@ -29,6 +29,7 @@ def perform
         instances = @instance_manager.filter_by(deployment, filter)
 
         ssh_info = instances.map do |instance|
+          begin
```

Comment:
Okay to re-indent even though you will lose git history (other pairs may feel differently)